src/types/index.ts

```diff
@@ -512,12 +512,27 @@ export type ZoneCapabilities =
       type: "HOT_WATER";
     };
 
-export type AwayConfiguration = {
-  type: ZoneType;
+export type HeatingAwayConfiguration = {
+  type: ZoneType & "HEATING";
   preheatingLevel: AwayConfigurationPreheatingLevel;
   minimumAwayTemperature: Temperature;
 };
 
+export type HotWaterAwayConfiguration = {
+  type: ZoneType & "HOT_WATER";
+  power: Power;
+  temperature: Temperature;
+};
+
+export type CoolingAwayConfiguration = Omit<TimeTableSettings, "isBoost"> & {
+  type: ZoneType & "AIR_CONDITIONING";
+};
```

Comment:
Will need someone to check these. As far as i can tell it is either as described above or the types above are nested inside a `settings` key. But I suspect this is correct. No way to know unless someone with cooling and hot water zones tries it.

Reply:
Yeah that is what I thought. And this way the new ones are consistent with the previous `HeatingAwayConfiguration` (which i can confirm is correct).